src/Microsoft.AspNet.Razor/Generator/Compiler/CodeBuilder/CSharp/Visitors/CSharpCodeVisitor.cs

```diff
@@ -515,5 +515,28 @@ public static CSharpCodeWriter RenderPreWriteStart(CSharpCodeWriter writer, Code
 
             return writer;
         }
+
+        /// <summary>
+        /// Renders the Write method for TagHelpers.
+        /// </summary>
+        /// <param name="writer"><see cref="CSharpCodeWriter"/> to which the output must be written.</param>
+        /// <param name="context"><see cref="CodeBuilderContext"/> containing the necessary data to be written.</param>
+        /// <returns><see cref="CSharpCodeWriter"/> with the written content.</returns>
+        public static CSharpCodeWriter RenderWriteForTagHelperContent(
+            CSharpCodeWriter writer, CodeBuilderContext context)
```

Comment:
why pass `writer` in?  if method were in `CSharpTagHelperCodeRenderer`, would use `_writer`.  here should use `Writer`.
nit; if you're wrapping, wrap all parameters to their own lines.